modules/nf-core/dragmap/align/dragmap-align.diff

```diff
@@ -1,19 +0,0 @@
-Changes in module 'nf-core/dragmap/align'
---- modules/nf-core/dragmap/align/main.nf
-+++ modules/nf-core/dragmap/align/main.nf
-@@ -2,10 +2,10 @@
-     tag "$meta.id"
-     label 'process_high'
- 
--    conda "bioconda::dragmap=1.3.0 bioconda::samtools=1.16.1 conda-forge::pigz=2.3.4"
-+    conda "bioconda::dragmap=1.2.1 bioconda::samtools=1.15.1 conda-forge::pigz=2.3.4"
```

Comment:
should we update samtools here as well, or wait for dragmap to become agreeable?

Reply:
I opened a PR over at biocontainers with updated samtools for both versions of dragmap

Reply:
thanks @matthdsm

Reply:
for me we can also update all mapper / samtools combis in another PR when the biocontainers are build